research/workflow-engines/index.md

```diff
@@ -0,0 +1,27 @@
+---
+title: Workflow engines
+authors: lbarcziova
+---
+
+- this research aims to describe workflow engines that could improve the state of the service, in particular
+  how the logic around job dependencies could be eased by usage of these
+
+- you can find separate files describing tools that were proposed to be researched more:
```

Comment:
Since you've spent a significant time on that, could you please add a conclusion with your opinion and suggest the next steps?

Reply:
sure! I firstly wanted to gather some feedback on what to extend and then in the end provide the conclusion, I will add this today

Reply:
Sure, thanks a lot!